Translation corrections (#272)
07th-mod/higurashi-console-arcs: Update/yoig_012.txt

			NULL, "\"I was an idiot.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s10/00/vt3a_yhead1006", 256, TRUE);
	OutputLine(NULL, "園崎魅音…頭首様に刃向かおうなどと…、本当に浅はかだった。",

> 			NULL, " It was shallow of my to turn my blade on Mion Sonozaki... the true head of the family.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " It was shallow of me to turn my blade on Mion Sonozaki... the true head of the family.", Line_WaitForInput);
```

//rそして、転がるように本邸の廊下に上がると、手をついて一歩…いや、ほんの少しずつ、奥の間へ向かってずるずると這って進んでいった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、転がるように本邸の廊下に上がると、手をついて一歩…いや、ほんの少しずつ、奥の間へ向かってずるずると這って進んでいった。",
			NULL, "Then, she began walking toward the exterior hallway of the main building... No, it might be more accurate to say she advanced at a crawling pace.", Line_Normal);
	ClearMessage();

//rその、彼女の這ったあとには、真っ赤な血がかすれた染みとなって残っている…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その、彼女の這ったあとには、真っ赤な血がかすれた染みとなって残っている…。",

> 			NULL, "The bright red blood splatter on her left faint trails in her wake...", Line_Normal);

Reviewer: ```suggestion
			NULL, "The bright red blood splattered on her body left faint trails in her wake...", Line_Normal);
```

//rだが…、そんな悩みなど、うまく生きようというただの打算に過ぎないことを思い知らされた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だが…、そんな悩みなど、うまく生きようというただの打算に過ぎないことを思い知らされた。",

> 			NULL, "However... I've had it drilled into me that those sorts of worries were nothing more than calculations known as living a good life.", Line_Normal);

Reviewer: ```suggestion
			NULL, "However... I was shown that those sorts of worries were nothing more than calculations for living a good life.", Line_Normal);
```

//r迷う必要も、どれが正しく、どれが楽かと悩む必要もない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "迷う必要も、どれが正しく、どれが楽かと悩む必要もない。",
			NULL, "There's no need to hesitate, to fret over which way is right, or worry about what would be easiest.", Line_Normal);
	ClearMessage();

//rただ、目の前のものに全力で当たっていく。そうして出来てくるのが、己の生き方だとすれば…、俺が雛見沢に来たのは宿命だったんだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ただ、目の前のものに全力で当たっていく。そうして出来てくるのが、己の生き方だとすれば…、俺が雛見沢に来たのは宿命だったんだ。",

> 			NULL, "I'll just face the things in front of me with all my strength. If I it was my lifestyle that enables me to do that... then it must've been fate that brought me to Hinamizawa.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I'll just face the things in front of me with all my strength. If my lifestyle is what enables me to do that... then it must've been fate that brought me to Hinamizawa.", Line_Normal);
```
